networking/tcp/tcp_keepalive/runtest.sh

```diff
@@ -40,6 +40,9 @@ fi
 
 rlJournalStart
     rlPhaseStartSetup
+       rlRun "systemctl stop firewalld" 0-255
```

Comment:
Shouldn't this be `rlServiceStop firewalld` instead? That way it will be compatible with systems that don't have `systemctl`.

However, I don't remember what happens when stopping the service fails... but I don't know how important it is.

Reply:
Thanks for your review, I changed the code with  rlServiceStop, please check again.
Thanks
Xiumei